src/components/Footer/Footer.js

```diff
@@ -1,6 +1,52 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
+import { FOOTER_TOP_LIST } from './data';
+import { FOOTER_BOTTOM_LIST } from './data';
```

Comment:
한군데서 가져오는 모듈이면 같이 작성할 수 있습니다.
```suggestion
import { FOOTER_TOP_LIST, FOOTER_BOTTOM_LIST } from './data';
```